addons/survey/static/src/js/survey_form.js

```diff
@@ -46,6 +53,38 @@ publicWidget.registry.SurveyFormWidget = publicWidget.Widget.extend({
     // Handlers
     // -------------------------------------------------------------------------
 
+    _onKeyPress: function (event) {
+        var self = this;
+        var aIndex = 'a'.charCodeAt(0);
+        var zIndex = 'z'.charCodeAt(0);
+        var code = event.keyCode;
+        var insideTextArea = this.$("textarea").is(":focus");
+        var $choiceInputs = this.$("input[type='radio'],input[type='checkbox']");
+        var isFieldsetDisabled = this.$("fieldset:disabled").length > 0;
+        // Handle Start / Next / Submit
+        if (code === 13 && !insideTextArea) {  // Enter : go Next
+            event.preventDefault();
+            this.$("button.btn-primary").click();
+        } else if (aIndex <= code < zIndex
+                && self.options.questionsLayout === 'page_per_question'
+                && $choiceInputs.length && !insideTextArea
+                && !isFieldsetDisabled) {
+            var keyIndex = code - aIndex;
+            if ($choiceInputs.length > 26) {
```

Comment:
Shouldn't it be made plus haut ?

First enter
Then more than 26 choices
then the other big condition